model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -252,7 +252,8 @@ case class MethodRepoConfigurationExport(
 case class WorkspaceListResponse(accessLevel: WorkspaceAccessLevel,
                                  workspace: Workspace,
                                  workspaceSubmissionStats: WorkspaceSubmissionStats,
-                                 owners: Seq[String])
+                                 owners: Seq[String],
+                                 public: Option[Boolean])
```

Comment:
i find it a little strange that we are only putting this attribute on the WorkspaceListResponse. It is not on the WorkspaceResponse. Do we think we will need it on the workspace itself?